packages/@lightningjs/ui-components/src/components/TabBar/Tab.d.ts

```diff
@@ -49,3 +74,5 @@ export default class Tab extends Surface {
   get _Icon(): lng.Component;
   get _Text(): lng.Component;
 }
+
```

Comment:
Looks good, but can you verify if you have added all the tags because I think content is missing in Tab and I don't see any tags in TabBar.d.ts file?

Reply:
minor one, remove type from import statement.